graphql_api/schema/custom/scalars.py

```diff
@@ -0,0 +1,28 @@
+from graphene.types.scalars import MIN_INT, MAX_INT
+from graphql.language.ast import IntValue
+from graphene.types.scalars import Scalar
+
+class BigInt(Scalar):
+    """
+    BigInt is an extension of the regular Int field
+        that supports Integers bigger than a signed
+        32-bit integer.
+    """
+    @staticmethod
+    def coerce_int(value):
+        try:
+            num = int(value)
+        except ValueError:
```

Comment:
@benjamineac this would only be needed if we already were using floats in some of our clients right (this used to be a workaround) . I think maybe we should drop it as it's bad to let users provide a float then drop the precision silently.